rpcs3/Emu/SysCalls/Modules/cellSysutil.cpp

```diff
@@ -224,12 +223,17 @@ s32 cellSysCacheMount(vm::ptr<CellSysCacheParam> param)
 {
 	cellSysutil.Warning("cellSysCacheMount(param=*0x%x)", param);
 
-	//TODO: implement
+	// TODO: implement
 	char id[CELL_SYSCACHE_ID_SIZE];
 	strncpy(id, param->cacheId, CELL_SYSCACHE_ID_SIZE);
 	strncpy(param->getCachePath, ("/dev_hdd1/cache/" + std::string(id) + "/").c_str(), CELL_SYSCACHE_PATH_MAX);
 	param->getCachePath[CELL_SYSCACHE_PATH_MAX - 1] = '\0';
-	Emu.GetVFS().CreateDir(std::string(param->getCachePath));
+	std::string directory = param->getCachePath;
+
+	if (!Emu.GetVFS().ExistsDir(directory))
```

Comment:
There's no point in trying to create a directory, if the directory with the given name already exists. Just prevents an unnecessary error from appearing in the log.